src/js/services/forms.js

```diff
@@ -151,6 +156,9 @@ export default App.extend({
       });
   },
   fetchFormStoreSubmission({ submission }) {
+    // NOTE: Remove after 2023/10/19
+    patchEvernorthHistory(submission);
```

Comment:
Does `fetchFormStoreSubmission` cover both a cloud draft and loading from local storage?

Reply:
yep, this is the only way a submission is preloaded into a form, right before it's sent to the form app.